README.md

```diff
@@ -731,6 +731,30 @@ O ExampleProviderTest mostrado precisa fornecer o estado de acordo com o arquivo
 
 ## <a id="sec-end-to-end-tests"></a>End-to-End Tests
 
+Testar sua aplicação por meio de sua interface de usuário é a maneira mais completa de testar sua aplicação. Os testes de IU orientados por webdriver descritos anteriormente são um bom exemplo de testes _end-to-end_.
```

Comment:
Escrever "testes de interface de usuário" ou "testes de UI" para manter o padrão com o restante do texto. Colocar a palavra webdriver em itálico.